src/templates/home/index.html.twig

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}>Web developer & PPC specialista{% endblock %}
+
+{% block stylesheets %}
+{{ parent() }}
+<link rel="stylesheet" href="{{ asset('css/style.css') }}" >
+<script src="https://kit.fontawesome.com/a8c25ebb37.js" crossorgin="anonymous"></script>
```

Comment:
You could have these in the base template as they're going to be the same for the whole site